src/Parser/VengeanceDemonHunter/Modules/PainChart/PainComponent.js

```diff
@@ -45,7 +45,7 @@ const PainComponent = ({ categories, abilities }) => {
                     {wasted || '0'}
                   </td>
                   <td style={{ width: '25%', color: 'orange' }}>
-                    {canBeImproved && !ability.noCanBeImproved && 'Can be improved.'}
+                    {wasted > 0 ? 'Can be improved.' : ''}
```

Comment:
Within JSX it's common to do `{wasted > 0 %% 'Can be improved.'}`

Reply:
I think the ternary conditional is more code-friendly. Almost every language has an ternary conditional in this exact same way. Putting the `{wasted > 0 %% 'Can be improved.}` is great because it's more objective, but `{wasted > 0 ? 'Can be improved.' : ''}` is easier to understand.

Reply:
I meant `{wasted > 0 && 'Can be improved.'}` which is common in dynamic languages